upf/upf_api.c

```diff
@@ -845,6 +845,54 @@ vl_api_upf_pfcp_policer_show_t_handler (vl_api_upf_pfcp_policer_show_t * mp)
   vl_api_send_msg (reg, (u8 *) rmp);
 }
 
+/* API message handler */
+static void
+vl_api_upf_pfcp_heartbeats_set_t_handler (vl_api_upf_pfcp_heartbeats_set_t *
+					  mp)
+{
+  vl_api_upf_pfcp_heartbeats_set_reply_t *rmp = NULL;
+  u32 retries, timeout = 0;
+  upf_main_t *sm = &upf_main;
+  int rv = 0;
+
+  retries = clib_net_to_host_u32 (mp->retries);
+  timeout = clib_net_to_host_u32 (mp->timeout);
+
+  rv = vnet_upf_pfcp_heartbeat_config (timeout, retries);
+
+  REPLY_MACRO (VL_API_UPF_PFCP_HEARTBEATS_SET_REPLY);
+}
+
+/* API message handler */
+static void
+vl_api_upf_pfcp_heartbeats_show_t_handler (vl_api_upf_pfcp_heartbeats_show_t *
+					   mp)
+{
+  vl_api_upf_pfcp_heartbeats_show_reply_t *rmp = NULL;
+  upf_main_t *sm = &upf_main;
+  pfcp_server_main_t *psm = &pfcp_server_main;
+  vl_api_registration_t *reg;
+
+  reg = vl_api_client_index_to_registration (mp->client_index);
+  if (!reg)
+    {
```

Comment:
`{` and `}` not needed for the single statement

Reply:
it's copied over from other places, so perhaps can leave it as-is, or correct everywhere